root/static/scripts/edit/externalLinks.js

```diff
@@ -527,13 +531,27 @@ export function parseRelationships(
 const protocolRegex = /^(https?|ftp):$/;
 const hostnameRegex = /^(([A-z\d]|[A-z\d][A-z\d\-]*[A-z\d])\.)*([A-z\d]|[A-z\d][A-z\d\-]*[A-z\d])$/;
 
+export function getUnicodeUrl(url: string): string {
+  if (!isValidURL(url)) {
+    return url;
+  }
+
+  const urlObject = new URL(url);
```

Comment:
This might not be supported using IE11 but (from [chat logs](https://chatlogs.metabrainz.org/brainzbot/metabrainz/msg/4729570/)):

> it seems to be provided by core-js
> https://github.com/zloirock/core-js#url-and-urlsearchparams